torch/_inductor/fx_passes/fuse_attention.py

```diff
@@ -536,6 +536,32 @@ def _sfdp_replacement_19(query, key, value, causal_mask, attn_mask, dropout_p):
     )
 
 
+def _sfdp_pattern_20(query, key, value, attn_mask):
+    # For huggingface bert
```

Comment:
please check if we can merge the logic with pattern_16 to avoid dedup.

Reply:
I tried merging pattern16 but I failed.
Pattern 16 cannot match this model.